src/config/dark-sites.config

```diff
@@ -151,6 +151,8 @@ spacespacespaaace.space
 speedtest.net
 steamcharts.com
 steamcommunity.com
+steam.tv
```

Comment:
Please, fix alphabetical order. I cannot make it myself, for some reason I have no access to your PR.

Reply:
That's because I deleted the my rep after the PR (I guess I need to stop doing that). And I believe the order is ok.

Reply:
`steam.tv` should go before `steamcharts.com`.

Reply:
Well, since I deleted my rep after doing pull request and cannot edit it, can you accept this pull request and make the change yourself?

Reply:
No, it fails a test.

Reply:
You don't have to create a repository, locate this file on GitHub, click Edit and GitHub will do everything for you.

Reply:
I deleted my edit and cannot change the pull request. Now this Pull request is basically read-only and can only be merged or not. That's why I says merge from "unknown repository". You can close this if you want and I will create another one (and not deleted it until after it's merged, I guess I learn a lesson here 🙂)